wrappers/s2i/python-conda/tests/pytorch-env/conda.yaml

```diff
@@ -0,0 +1,9 @@
+channels:
+  - defaults
+dependencies:
+  - python=3.7.0
+  - pip
+  - pytorch
+  - pip:
+      - seldon_core
```

Comment:
Should we add limits on major versions?

Reply:
Good shout! Let's learn from our own mistakes... :joy: